snippets/csharp/programming-guide/strings/string_program_1.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace StringCompare
+{
+    class Program
+    {
+        static void Main()
+        {
+            string jupiterFirst = "Jupiter";
```

Comment:
That file should be enough: use snippet tags to bound two relative code fragments.
@BillWagner should the *.csproj* file be checked in as well?

Reply:
Yes, checkin the csproj. The CI will build and validate the samples.

Reply:
csproj checked in. Included snippet tags.